lib/ngx/ocsp.lua

```diff
@@ -119,7 +119,9 @@ function _M.validate_ocsp_response(resp, chain, max_errmsg_len)
     if rc == FFI_OK then
         local next_update = tonumber(next_update_p[0])
         if next_update == 0 then
-          next_update = nil
+            next_update = nil
+        else
+            next_update_p[0] = 0
```

Comment:
Would you please put this line before line 114?

Reply:
sure.